app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -432,8 +430,8 @@ class TabTrayView(
     private fun setupListTabView() {
         view.tabsTray.apply {
             layoutManager = LinearLayoutManager(container.context).apply {
-                reverseLayout = true
-                stackFromEnd = true
+                reverseLayout = false
```

Comment:
I think `false` is the default value for these properties so we don't really need these?
For gridView we would have the same behavior and for it we are not explicit about this behavior.
If there aren't any other specific changes for listView maybe we don't need this method at all?